### ISSUE TITLE
Plugin fails to install on cordova >7

### ISSUE THREAD

**Issue by mark-veenstra** [opentok/cordova-plugin-opentok#33]
`user@user-laptop:~/Documenten/Projects/open-tok-poc$ cordova plugin add https://github.com/opentok/cordova-plugin-opentok --save`

```
(node:12030) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Failed to fetch plugin https://github.com/opentok/cordova-plugin-opentok via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Failed to get absolute path to installed module
```

**Comment by msach22** [comment]
@mark-veenstra so I just added the plugin on Cordova 7 and I did not get this error. Are you getting this error consistently?

**Comment by mark-veenstra** [comment]
@msach22 Well I did had the same error multiple times. But I found out that within the project itself (not plugin) the `package-lock..json` is the main reason why it is failing. Not sure why yet, but I have to remove it when I upgrade from `cordova@6.5.0` to `cordova@8.0.0`.

So the main error is the `package-lock.json`. But still the package name in the `package.json` should be the same as the plugin id from the `plugin.xml`.
